docs/06-concepts/02-models.md

```diff
@@ -161,72 +161,98 @@ extension MyExtension on MyClass {
 |[**unique**](database/indexing)|Boolean flag to make the entries unique in the database.                                         |✅|||
 |[**default**](#default-values)|Sets the default value for both the model and the database. This keyword cannot be used with **relation**.                                                     |✅|||
 |[**defaultModel**](#default-values)|Sets the default value for the model side. This keyword cannot be used with **relation**.                                                   |✅|||
```

Comment:
Question: I suppose this has not been added to exceptions, but doesn't it make sense to also add this feature for the exception type? It should be rather trivial to add too.

Reply:
I think this is better to it in another pr.